tests/integration/questionnaire/test_questionnaire_page_titles.py

```diff
@@ -0,0 +1,116 @@
+from extraction import Extractor
+
+from tests.integration.create_token import create_token
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestQuestionnairePageTitles(IntegrationTestCase):
+
+    def test_should_have_question_in_page_title_when_loading_introduction(self):
+        # Given
+        token = create_token('final_confirmation', 'test')
+        resp = self.client.get('/session?token=' + token.decode(), follow_redirects=True)
+
+        # When
+        extraction = Extractor().extract(resp.get_data(True))
```

Comment:
when move this into a helper in the base class like `get_page_title(response)` so we can start to build more convenience functions in there? We should think